app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html

```diff
@@ -66,21 +66,30 @@ <h2 class="heading-medium" id="subsection-title">
 @mainTemplate(title = title, userType = None, withSignOutLink = false, sidebar = Some(sidebarContent)) {
 
     <h1 class="heading-xlarge">@title</h1>
-    <h2 class="heading-medium">@messages("landingPage.contents.h2")</h2>
-    <ol>
-        <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet1", "#overview"))</li>
-        <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet2", "#howTo"))</li>
-        <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet3", "#beforeYouStart"))</li>
-        <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet4", "#ifAgent"))</li>
-        <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet5", "#ifTrust"))</li>
-        <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet6", "#capasitorOrPersonalRepresentative"))</li>
-    </ol>
-    <h2 class="heading-medium" id="overview">@messages("landingPage.intro.h2")</h2>
+    @if(capacitorsAndPersonalRepresentativesJourneyEnabled) {
+        <h2 class="heading-medium">@messages("landingPage.contents.h2")</h2>
+        <ol>
+            <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet1", "#overview"))</li>
+            <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet2", "#howTo"))</li>
+            <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet3", "#beforeYouStart"))</li>
+            <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet4", "#ifAgent"))</li>
+            <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet5", "#ifTrust"))</li>
+            <li class="list-item--dashed">@Html(messages("landingPage.contents.list.bullet6", "#capasitorOrPersonalRepresentative"))</li>
+        </ol>
+        <h2 class="heading-medium" id="overview">@messages("landingPage.intro.h2")</h2>
+    } else {
+        @Html("")
```

Comment:
do we not want the contents bit to still be there if capacitors/personal reps are disabled?

Reply:
For the individuals, there was no contents bit initially. So we are behaving like the prev.

Reply:
ah ok, I forgot that I'm looking at a diff with the new content on the left already